Homework 6

LukaVukovic2 opened the pull request:
Assignment 1, Assignment 2, Assignment 3
- add Roboto fonts
- custom style Button, Card, Heading and Input component
- create sidebar navigation appropriate for mobile devices
- responsive web and mobile layout

LukaVukovic2 committed:
Add fonts and style components

LukaVukovic2 committed:
Style reviews page

LukaVukovic2 committed:
Adjust application theme

LukaVukovic2 committed:
Add Heading variants

-add variants for web, tablet and mobile
-enable upload image preview on "MyProfile" page
-add breakpoints

LukaVukovic2 committed:
Create mobile layout for show list page

LukaVukovic2 committed:
Create mobile layout for register/login page

LukaVukovic2 committed:
Add Drawer component

-create navigation appropriate for smaller screen sizes
-toggle between Sidebar navigation and Drawer navigation after
breakpoint
-create NavigationList component to reuse common content
-trigger Drawer via Hamburger icon
-close Drawer after clicking navigation item

LukaVukovic2 committed:
Design fully responsive layouts

LukaVukovic2 committed:
Handle show/reviews data change with SWR

LukaVukovic2 committed:
Branch clean up

LukaVukovic2 committed:
Update tests

-LoginForm test
-ReviewItem test
-ReviewList test

LukaVukovic2 committed:
Resolve conflicts

ante-pesorda commented on homework-4/src/app/(shows)/my-profile/page.tsx:
🔥

ante-pesorda commented on homework-4/src/components/shared/Header/Header.tsx:
Pogledati kako rade Show/Hide komponente u Chakri

ante-pesorda commented on homework-4/src/components/shared/ShowCard/ShowCard.tsx:
```diff
@@ -20,16 +21,28 @@ export default function ShowCard({ show }: { show: IShow }) {
-          {!show.average_rating && " Not rated yet"}
-        </Text>
+      <CardBody
+        color="purple2"
```
Je li umjesto color2 možda bolje napraviti ljubičaste boje s nijansama (100, 200, 300...)

ante-pesorda reviewed (approved):
Header na mobitelu bih stavio kao sticky

LukaVukovic2 committed:
Improve styles

-sticky header on mobile devices
-show/hide Chakra component instead of ternary conditional operator
-shades of colors with .x00

ante-pesorda reviewed (commented):
Tebi aplikacija sad radi kad je pokreneš?
Folder 4, branch 6

LukaVukovic2 commented:
> Tebi aplikacija sad radi kad je pokreneš? Folder 4, branch 6

Da
![image](https://github.com/user-attachments/assets/bc45f196-404f-4718-be1f-edf6bc8cecc6)
